plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -118,7 +118,8 @@ public LibraryClasspathContainer loadContainer(IJavaProject javaProject, IPath c
     }
   }
 
-  public void saveLibraryIds(IJavaProject javaProject, List<String> libraryIds)
+  // todo containerPath argument is not used
```

Comment:
Then why add the argument?

Reply:
To match `loadLibraryIds()`, and in symmetry with the loadContainer()/saveContainer() methods.

I'm not sure whether we should be using the container path any more, or if there's any value in trying to support multiple containers, or different containerIds (e.g., we currently specify 'appengine').